js_modules/dagit/packages/core/src/pipelines/PipelineExplorerRoot.tsx

```diff
@@ -106,22 +100,13 @@ export const PipelineExplorerContainer: React.FC<{
           );
         }
 
-        const repositoryAssets =
-          repositoryOrError.__typename === 'Repository' ? repositoryOrError.assetNodes : [];
-        const isAssetGraph = result.solidHandles.some((handle) =>
-          repositoryAssets.some(
-            (asset) =>
-              asset.opName === handle.handleID && asset.jobName === explorerPath.pipelineName,
-          ),
-        );
+        const jobAssets =
+          pipelineOrError.__typename === 'Pipeline' ? pipelineOrError.assetNodes : [];
+        const isAssetGraph = jobAssets.length > 0;
```

Comment:
It seems like this should use isAssetJob here?

Reply:
👍